docs/src/components/typography/font-size.module.css

```diff
@@ -15,9 +15,9 @@
 }
 .sizeLabel {
   color: var(--appColorsTextIconLow);
-  font-size: var(--psTypeFontSizeXSmall);
+  font-size: var(--psTypeFontSizeBase);
 }
-.sizeVar {
+.variable {
```

Comment:
`variable` is just fine here. The module is small and single-use.

Just some context as to why `sizeVar` was used and why you might see this pattern elsewhere in the code:

CSS is usually in a global context. Modules change that.  This is a module, so it's a context limited to font-size stuff.  But in global context, we'd group things together and show hierarchy through naming conventions such as BEM. `sizeVar` is not BEM, but it is similar in that the B)lock could be the `size` and then E)lement could be the `Var`. Together: `sizeVar`. Precisely, no. But, you do get some hierarchy info about the DOM that way: `sizes > size > sizeLabel & sizeVar`.

Reply:
I renamed the variables to better scope. Another case where some layout utility components would eliminate the need for a lot of these docs sub components.

Reply:
Yeah, if you're seeing a pattern that we repetitively document, a utility here would be great.